addons/context_actions/functions/fnc_teleportPlayers.sqf

```diff
@@ -21,8 +21,17 @@ private _players = _this select {isPlayer _x};
     params ["_successful", "_players", "_posASL"];
 
     if (_successful) then {
-        {
-            _x setPosASL _posASL;
-        } forEach _players;
+        private _curatorHovered = curatorMouseOver;
+        _curatorHovered params ["_type", "_entity"];
+
+        if (_type isEqualTo "OBJECT") then {
+            private _result = [_players, _entity] call EFUNC(common,teleportToVehicle);
+            if (_result) exitWith {};
+            [localize "str_3den_notifications_vehiclefull_text"] call EFUNC(common,showMessage);
```

Comment:
The message is: "No space in target vehicle" but I think this will show when there are not enough empty positions to fill all units (trying to teleport 2 people into a vehicle with 1 free seat). Maybe we should use a custom string?

Also, don't need `localize`, `showMessage` handles that.

Reply:
If we want custom string we might use something like "Not enough space in target vehicle: %1 needed - %2 available" so Zeus will get an information about target vehicle cargo space which might be useful. Zeus will know exactly how much space there is instead making blind guesses or putting players one by one.

Reply:
👍 Sounds good.